avocado/plugins/run.py

```diff
@@ -128,10 +128,13 @@ def configure(self, parser):
 
         if multiplexer.MULTIPLEX_CAPABLE:
             mux = parser.add_argument_group('multiplexer use on test execution')
-            mux.add_argument('-m', '--multiplex-files', nargs='*',
+            mux.add_argument('-m', '--multiplex', nargs='*', dest='multiplex_files',
                              default=None, metavar='FILE',
                              help='Location of one or more Avocado multiplex (.yaml) '
                              'FILE(s) (order dependent)')
+            mux.add_argument('--multiplex-files', nargs='*',
+                             default=None, metavar='FILE',
+                             help='DEPRECATED: please use --multiplex instead')
```

Comment:
Abusing type is bad. Using action is fine.

My personal preference is to not add custom code that will eventually have to be removed. Also, I personally don't like printing the deprecation messages.

IMHO, users should be given notice (release notes, help messages) and a few releases to adapt. Then, on removal of the deprecated option, the *automatic error* will kick in.

These are my firm beliefs, others may feel differently. @adereis care to comment?